src/main/scala/ch/epfl/bluebrain/nexus/kg/async/RevisionedValue.scala

```diff
@@ -1,17 +1,9 @@
 package ch.epfl.bluebrain.nexus.kg.async
 
-import akka.cluster.ddata.LWWRegister.Clock
-
 /**
   * A value with an attached revision corresponding to the value.
   *
   * @param rev   the value revision
   * @param value the value
   */
 final case class RevisionedValue[A](rev: Long, value: A) extends RegisteredValue[A]
```

Comment:
I would recommend renaming `RegisteredValue[A]` to `RegisterValue[A]`.

Reply:
And possibly make it a sealed trait while moving the sub types in the same file.